README.md

```diff
@@ -1,27 +1,35 @@
 <div align="center">
 <a href="https://era.zksync.io/docs/tools/testing/era-test-node.html">
 
-![era-test-node](./.github/assets/era_test_node_banner_light.png#gh-light-mode-only)
-![era-test-node](./.github/assets/era_test_node_banner_dark.png#gh-dark-mode-only)
+![anvil-zksync](./.github/assets/era_test_node_banner_light.png#gh-light-mode-only)
+![anvil-zksync](./.github/assets/era_test_node_banner_dark.png#gh-dark-mode-only)
 </a>
 
   </div>
 
-# 🚀 zkSync Era In-Memory Node 🚀
+# 🚀 anvil-zksync 🚀
+
+# ⚠️ Repository Renamed
+
+This repository has been renamed from **era-test-node** to **anvil-zksync**. All references to the previous name have been updated to reflect this change.
```

Comment:
Probably it would work better as:

```
> [!IMPORTANT]  
> This repository has been renamed from **era-test-node** to **anvil-zksync**. All references to the previous name have been updated to reflect this change.
```

which will look like

> [!IMPORTANT]  
> This repository has been renamed from **era-test-node** to **anvil-zksync**. All references to the previous name have been updated to reflect this change.